react/features/welcome/components/WelcomePage.web.js

```diff
@@ -36,6 +137,17 @@ export default class WelcomePage extends Component {
     }
 
     /**
+    * Sets input element as property of class.
+    *
+    * @param {HTMLInputElement} input - input element to be set.
+    * @returns {void}
+    * @private
+    **/
+    _setInput(input) {
+        this.roomNameInput = input;
+    }
```

Comment:
I'm afraid I find the business with _setInput and _roomNameInput unnecessary and I'll remove them.